### PR TITLE
update partners, some other random cleanup

### PR DESCRIPTION
**Pull request by critzo**
- add wixi logo
- remove proj connect logo
- removed old observatory code
- update observatory page template
- update travis and gems

@georgiamoon PTAL? Previewable on my fork: https://critzo.github.io/m-lab.github.io/who/

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/357)

### CODE REVIEW

**Comment by georgiamoon** [comment]
@critzo I noticed that you are deleting the observatory sub-module here. Won't that break historic links that where embedded in documents?

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: ruby
 rvm:
-  - 2.2.3
+  - 2.4.1
 script:
-  - ./_tests/travis-checks
+  - ./_tests/travis-checks --disable-external
```
Comment (georgiamoon) [inline .travis.yml]: This is causing the travis build to break. Can you check on this?

**Comment by critzo** [comment]
@georgiamoon This build is working now. Thoughts on the observatory sub-module?

**Comment by georgiamoon** [comment]
@critzo I guess my thought is that if we are removing the submodule, we should change it to re-direct to the viz page, rather than making that an extra click through. Before we had been trying to maintain that page for any potential links to it, but in lieu of that, I'd vote for a redirect. Thoughts?

**Comment by critzo** [comment]
@georgiamoon I agree about the adding the redirect. I'll add that and ping when done.

**Comment by critzo** [comment]
@georgiamoon redirect has been added

**Pull request merged**